### DISCUSSION TITLE
Plugin help - jsPsych 7.3 and Pavlovia

### DISCUSSION THREAD

**Discussion by aschutz32** [jspsych/jsPsych#2955] [Q&A] [answered]
I am not a programmer and am new to coding. I coded my entire experiment with jsPsych version 7.3 because I initially intended to use Heroku or Cognition.run to host my experiment online. However, due to the cost of these platforms I am now turning to Pavlovia. 

It took a really long time to code my experiment with jsPsych because I am new to coding, and because there are a lot of files. I am trying to integrate my experiment with Pavlovia but the plugin appears to be an older version and I am unable to find a version that will work with jsPsych 7.3. I have been trying to reference the [migration instructions](https://github.com/jspsych/jspsych-contrib/blob/main/packages/plugin-template/index.js) for updating plugins, but I am feeling a bit overwhelmed and not sure if I am doing it right. 

[This is the 7.0 version Pavlovia plugin](https://lib.pavlovia.org/jspsych-7-pavlovia-2022.1.1.js). It does not work when I try to use it as is, which is why I am resorting to re-making it. 

Any and all thoughts on this issue are welcome.

Thank you!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @aschutz32,

The pavlovia team created that plugin and they may be the best folks to get help from.

If you are looking for a free way to run a jsPsych experiment we just made DataPipe - https://pipe.jspsych.org. It might suit your needs if you can't get the pavlovia plugin to work.

**Reply by aschutz32** [reply]
Thank you! I am looking into DataPipe - thanks for the suggestion!
